README.md

```diff
@@ -197,3 +207,16 @@ Object with the default options for the notifications
 
 ## defaultOptionsByType
 Object with the default options for the notifications for specific types
+
+## Building
+
+_OS X users, ensure you have `wget` installed (available in `brew`)_
+
+To run the tests:
+
+```bash
+$ ./workaround.sh
```

Comment:
When I run `make test`  get this:

```bash
node start_test.js
I20150106-18:03:52.903(8)? test-in-console listening
starting testing...

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:980:11)
    at Process.ChildProcess._handle.onexit (child_process.js:771:34)
make: *** [test] Error 8
```

On 06/01/2015 17:45, Marco de Jongh wrote:
> |meteor test-packages ./|

-- 
Ramesh Nair

Freelance Web Developer

www.hiddentao.com
www.github.com/hiddentao